memoized_lcsh.json

```
@@ -410,5 +410,52 @@
   "Viral Proteins": "https://id.loc.gov/authorities/subjects/sh94006884",
   "Cytology": "https://id.loc.gov/authorities/subjects/sh99005436",
   "Lincoln Park (Chicago, Ill.)": "https://id.loc.gov/authorities/subjects/sh89006458",
-  "Piazza dei Miracoli (Pisa, Italy)": "https://id.loc.gov/authorities/subjects/sh2008002257"
+  "Piazza dei Miracoli (Pisa, Italy)": "https://id.loc.gov/authorities/subjects/sh2008002257",
+  "Skin--Inflammation": "http://id.loc.gov/authorities/subjects/sh85123181",
```

All of these should be with the `https` form e.g. 

```suggestion
  "Skin--Inflammation": "https://id.loc.gov/authorities/subjects/sh85123181",
```

Review terms below to use https all the time.